kolibri/plugins/style_guide/assets/src/views/shell/component-docs.vue

```diff
@@ -1,26 +1,13 @@
 <template>
 
-  <div class="component-template">
+  <div>
 
-    <h1>{{ api.name }}</h1>
+    <p>Import using: <code>{{ importString }}</code></p>
 
+    <p v-if="api.description">{{ api.description }}</p>
 
-    <p>{{ api.description }}</p>
-
-    <!--
-    <h2>Status</h2>
-    <p v-if="status === 'complete'" class="status-complete">Fully implemented</p>
-    <p v-else class="status-incomplete">Not fully implemented</p>
-    -->
-
-    <h2 id="api">API</h2>
-
-    <h3>Import</h3>
-    <code>{{ importString }}</code>
-
-
-    <template v-if="api.props.length">
-      <h3>Props</h3>
```

Comment:
The headings were problematic because we don't have control over the context in which the components doc can is used. In particular, in the navbar I used `<h3>` headings for each of the two components: https://github.com/learningequality/kolibri/pull/1925/files#diff-6729e6473dad99dd337ee6f12c7ba7b6R3

So then the `<h3>`s inside the component documentation aren't properly nested anymore...